core/lib/spree/core/importer/order.rb

```diff
@@ -52,48 +50,43 @@ def self.create_shipments_from_params(shipments_hash, order)
 
           line_items = order.line_items
           shipments_hash.each do |s|
-            begin
-              shipment = order.shipments.build
-              shipment.tracking       = s[:tracking]
-              shipment.stock_location = Spree::StockLocation.find_by_admin_name(s[:stock_location]) || Spree::StockLocation.find_by_name!(s[:stock_location])
-
-              inventory_units = s[:inventory_units] || []
-              inventory_units.each do |iu|
-                ensure_variant_id_from_params(iu)
-
-                unit = shipment.inventory_units.build
-                unit.order = order
-
-                # Spree expects a Inventory Unit to always reference a line
-                # item and variant otherwise users might get exceptions when
-                # trying to view these units. Note the Importer might not be
-                # able to find the line item if line_item.variant_id |= iu.variant_id
-                unit.variant_id = iu[:variant_id]
-                unit.line_item_id = line_items.select do |l|
-                  l.variant_id.to_i == iu[:variant_id].to_i
-                end.first.try(:id)
-              end
+            shipment = order.shipments.build
+            shipment.tracking       = s[:tracking]
+            shipment.stock_location = Spree::StockLocation.find_by_admin_name(s[:stock_location]) || Spree::StockLocation.find_by_name!(s[:stock_location])
```

Comment:
This line actually seems weird. Why do we find the stock location by find the stock location by finding the stock location name?